README.md

```diff
@@ -44,22 +56,22 @@ serviced host add HOST:PORT RESOURCE_POOL
 
 Dev Environment
 ---------------
-Serviced is written in go. To install go, download go v1.4 from http://golang.org.
-Untar the distribution to /usr/local/go. If you use a different location for go, you
+Serviced is written in go. To install go, download `go v1.14.4` from http://golang.org.
+Untar the distribution to `/usr/local/go`. If you use a different location for go, you
 must set GOROOT. See the http://www.golang.org for more information. Ensure that
-$GOROOT/bin is in you path.
+`$GOROOT/bin` is in you path.
```

Comment:
There is an alternative installation process on Ubuntu: the snap package.
```bash
$ sudo snap install --classic --channel=1.14/stable go
```

Reply:
Yes but we use not only ubuntu

Reply:
Correct, but since our most common development environment is Ubuntu, I think it's worth mentioning that developers on Ubuntu can take advantage of the snap package.

Reply:
Agree, added